src/sparseml/pytorch/sparsification/pruning/modifier_pruning_base.py

```diff
@@ -117,10 +117,16 @@ def __init__(
         log_types: Union[str, List[str]] = None,
         global_sparsity: bool = False,
         allow_reintroduction: bool = False,
+        explicit_kwargs: Optional[List[str]] = ["params"],
```

Comment:
`explicit` doesn't give us any extra information here, could we rename to `parent_class_kwarg_names` or something similar